websites/models.py

```diff
@@ -256,11 +256,11 @@ def url_path_from_metadata(self, metadata: dict | None = None):
                     content = self.websitecontent_set.filter(type=section_type).first()
                     if content:
                         value = get_dict_field(content.metadata, section_field)
-                if not value:  # noqa: SIM108
+                if not value:
                     # Incomplete metadata required for url
                     value = section
                 else:
-                    value = slugify(value.replace(".", "-"))
+                    value = slugify(value.replace(".", "-"), allow_unicode=True)
```

Comment:
The Studio UI doesn't allow Unicode characters in the url_path. I wonder if we should keep `allow_unicode=False` here since Unicode characters are invalid for this field.

<img width="673" alt="Screenshot 2024-02-07 at 5 13 45 PM" src="https://github.com/mitodl/ocw-studio/assets/71316217/86200a3b-b241-4f47-852e-4719e871397b">